config/docker/php/Dockerfile

```diff
@@ -1,11 +1,31 @@
-FROM 4xxi/php-pgsql:flex
+FROM php:7.2-fpm-alpine
 
-# To add additional packages and php extensions uncomment the following lines and add required dependencies
-# Check https://hub.docker.com/_/php/ for examples and documentation
+COPY ./symfony.ini /usr/local/etc/php/conf.d/
+COPY ./symfony.pool.conf /usr/local/etc/php-fpm.d/
 
-#RUN apt-get update && apt-get install -y \
-#    lib_name_1 \
-#    lib_name_2 \
-#    && docker-php-ext-install \
-#    ext_name_1 \
-#    ext_name_2
\ No newline at end of file
+RUN apk add --no-cache \
+    git \
+    zip \
+    icu-dev \
+    zlib-dev \
+    postgresql-dev
+
+RUN docker-php-ext-install \
+    intl \
+    opcache \
+    pdo \
+    pdo_pgsql \
+    zip
+
+RUN EXPECTED_SIGNATURE="$(wget -q -O - https://composer.github.io/installer.sig)" \
```

Comment:
Package installation can be replaced by copying a binary file from the official repository.
`COPY --from=composer /usr/bin/composer /usr/bin/composer`

Reply:
Yes, I agree
This will help speed up the installation of dependencies and container assembly.